CMakeLists.txt

```diff
@@ -306,7 +306,12 @@ set(LIBNETDATA_FILES
         libnetdata/url/url.c
         libnetdata/url/url.h
         libnetdata/socket/security.c
-        libnetdata/socket/security.h)
+        libnetdata/socket/security.h
```

Comment:
I'm pretty certain you need to add something to this file for the json-c lib. cmake fails with a bunch of errors.

Reply:
My VM crashed when I was testing it, I am creating it again to confirm that I fixed.